src/web/mjs/connectors/ComicBunch.mjs

```diff
@@ -0,0 +1,64 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class ComicBunch extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'comicbunch';
+        super.label = 'コミックバンチ (ComicBunch)';
+        this.tags = [ 'manga', 'japanese' ];
+        this.url = 'https://www.comicbunch.com';
+    }
+
+    async _getMangaFromURI(uri) {
+        const request = new Request(uri, this.requestOptions);
+        const id = uri.pathname;
+        const title = (await this.fetchDOM(request, '#comics > h2'))[0].textContent.trim();
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        const mangaList = [];
+        const uri = new URL('/comics', this.url);
+        const request = new Request(uri, this.requestOptions);
+        let data = (await this.fetchDOM(request, 'select[name="sort"] option')).filter(item => item.value != '').map(item => item.value.substr(3));
+        for(var i = 0; i < data.length; i++) {
+            let mangas = await this._getMangasFromPage(data[i]);
+            mangaList.push(...mangas);
+        }
+        return mangaList;
+    }
+
+    async _getMangasFromPage(page) {
+        const uri = new URL('/comics/' + page, this.url);
+        const request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, '#comics_list > dl');
+        return data.filter(element => element.querySelector('dd a').className.includes('read_more')).map(element => {
```

Comment:
May use CSS matcher `dd a[class*="read_more"]` instead of `className.includes('read_more')`

Reply:
Fixed